test/unit/multiversion_test.py

```diff
@@ -84,27 +84,27 @@ def version_info(tmp_path):
     )
 
 
-def test_tags_property(version_info):
+def test_tags_property(version_info):  # pylint: disable=redefined-outer-name
```

Comment:
Can you disable this for the complete file to avoid repeating the comment?

Reply:
That is possible but not desired imho ;)